experimental/dds/ot/ot/src/ot.ts

```diff
@@ -18,7 +18,7 @@ import {
 } from "@fluidframework/shared-object-base";
 
 interface ISequencedOpInfo<TOp> {
-	client: string;
+	client: string | null;
```

Comment:
I'm not super familiar with this package, but similar to other PR consider whether null is really a valid thing for this object to receive and maybe assert instead.